jekyll/_cci2/server/installation/phase-3-execution-environments.adoc

```diff
@@ -205,10 +251,27 @@ ifndef::env-aws[]
 ==== GCP autoscaler service account
 Create a service account for Nomad Autoscaler. You may take **one** of the following approaches:
 
-* The CircleCI link:https://github.com/CircleCI-Public/server-terraform/tree/main/nomad-gcp[nomad module] can create a service account using link:https://circleci.com/docs/2.0/server-3-install-prerequisites/index.html#enabling-workload-identity-in-gke[Workload Identity] and out the email. Set the variables `nomad_auto_scaler = true` and `enable_workload_identity = true`.
 
+*Option 1 - CircleCI creates the secret
 * The CircleCI link:https://github.com/CircleCI-Public/server-terraform/tree/main/nomad-gcp[nomad module] can create a service account and output a file with the JSON key. For this option, set the variable `nomad_auto_scaler = true`. You may reference the examples in the link for more details. The created service account key will be available in a file named `nomad-as-key.json`.
 
+
+*Option 2 - Use Workload Identity
+* The CircleCI link:https://github.com/CircleCI-Public/server-terraform/tree/main/nomad-gcp[nomad module] can create a service account using link:https://circleci.com/docs/2.0/server-3-install-prerequisites/index.html#enabling-workload-identity-in-gke[Workload Identity] and out the email. Set the variables `nomad_auto_scaler = true` and `enable_workload_identity = true`.
+
+
+*Option 3 - Create the Secret yourself
+When creating the secret manually, an additional field is required - see the below example.
+
+[source, shell]
+----
+# Base64 encoded additional configuration field
+ADDITIONAL_CONFIG=dGFyZ2V0ICJnY2UtbWlnIiB7CiAgZHJpdmVyID0gImdjZS1taWciCiAgY29uZmlnID0gewogICAgY3JlZGVudGlhbHMgPSAiL2V0Yy9ub21hZC1hdXRvc2NhbGVyL2NyZWRzL2djcF9zYS5qc29uIgogIH0KfQo=
+kubectl create secret generic nomad-autoscaler-secret \
+  --from-literal=gcp_sa.json=<service-account> \
+  --from-literal=secret.hcl=$ADDITIONAL_CONFIG
+----
+
 * You may create a Nomad GCP service account manually. The service account will need the role `compute.admin`. It will also need the role `iam.workloadIdentityUser` if using link:https://circleci.com/docs/2.0/server-3-install-prerequisites/index.html#enabling-workload-identity-in-gke[Workload Identity]. This step is only required if you choose not to create the service account using Terraform.
```

Comment:
@circlery is this still one of the options? We have three options listed above, is this option 4? Or is it a duplicate?

Reply:
Hmm, I don't think I wrote this, but it's more of a note to the option where you provide a service account, since it's got nothing to do with actually providing the value to the values.yaml file, and more to do with how you create those values before hand - if that makes any kind of sense.

I reckon it's still relevant, but maybe lets keep it as a footnote of option 3, since I think that's the only place it applies.

Reply:
OK thanks. I'll add it into option three and if it doesn't make sense I'm sure I'll get feedback about it